pages/in-vitro-systems/[id].js

```diff
@@ -204,20 +204,20 @@ export async function getServerSideProps({ params, req, query }) {
     return {
       props: {
         inVitroSystem,
-        biosampleTerm,
+        sampleTerms,
         diseaseTerms,
         documents,
         donors,
-        source,
+        sources,
         treatments,
         pooledFrom,
         partOf,
         targetedSampleTerm,
         biomarkers,
         pageContext: {
-          title: `${biosampleTerm ? `${biosampleTerm.term_name} — ` : ""}${
-            inVitroSystem.accession
-          }`,
+          title: `${
+            sampleTerms.length > 0 ? `${sampleTerms[0].term_name} — ` : ""
+          }${inVitroSystem.accession}`,
```

Comment:
Here, you know `sampleTerms` contains something guaranteed, you don’t need to check for a length of zero. Just use `${inVitroSystem.sample_terms[0].term_name} — ${inVitroSystem.accession}`

Reply:
Done